tests/visualization_tests/test_contour.py

```diff
@@ -79,252 +108,443 @@ def fail_objective(_: Trial) -> float:
         plot_contour(study, ["optuna", "Optuna"])
```

Comment:
This error is raised within `_get_contour_info` so I think it should also be caught in `test_get_contour_info`.

Reply:
This error seems to be already caught in `test_get_contour_info_error`, so I think we can simply remove this test.